src/components/single/CameraSwitcher.vue

```diff
@@ -404,6 +404,11 @@ export default defineComponent({
           break;
       }
 
+      epcList.push(0xE4);
+      edtList.push([]);
+      epcList.push(0xD3);
+      edtList.push([]);
```

Comment:
@justdecodeme  As I commented earlier, you can't directly change epcList or so as it will affect the set-property function as well. You have to manage it at line #440 in line.